lib/rdoc/generator/json_index.rb

```diff
@@ -175,7 +175,7 @@ def generate_gzipped
     debug_msg "Writing gzipped search index to %s" % outfile
 
     Zlib::GzipWriter.open(outfile) do |gz|
-      gz.mtime = File.mtime(search_index_file)
+      gz.mtime = 1 # make output reproducible
```

Comment:
I think the `mtime` should be updated because the `search_index.js.gz` is generated dynamically. Please use SOURCE_DATE_EPOCH.